x-pack/plugins/reporting/public/management/components/report_info_flyout_content.tsx

```diff
@@ -95,93 +97,99 @@ export const ReportInfoFlyoutContent: FunctionComponent<Props> = ({ info }) => {
       }),
       description: info.size?.toString() || NA,
     },
-    {
-      title: i18n.translate('xpack.reporting.listing.infoPanel.attemptsInfo', {
-        defaultMessage: 'Attempts',
+
+    hasScreenshot && {
+      title: i18n.translate('xpack.reporting.listing.infoPanel.dimensionsInfoHeight', {
+        defaultMessage: 'Height (in pixels)',
+      }),
+      description: info.layout?.dimensions?.height || UNKNOWN,
```

Comment:
Let's round the number, since it can be an irrational number
```suggestion
      description:
        info.layout?.dimensions?.height != null
          ? Math.ceil(info.layout.dimensions.height)
          : UNKNOWN,

```